projects/storefrontapp/src/app/spartacus/features/digital-payments-feature.module.ts

```diff
@@ -1,14 +1,21 @@
 import { NgModule } from '@angular/core';
-import { provideConfig, I18nConfig } from '@spartacus/core';
-import { DigitalPaymentsModule } from '@spartacus/digital-payments';
+import { CHECKOUT_FEATURE } from '@spartacus/checkout/root';
+import { I18nConfig, provideConfig } from '@spartacus/core';
 import {
   dpTranslationChunksConfig,
   dpTranslations,
-} from '@spartacus/digital-payments';
+} from '@spartacus/digital-payments/assets';
 @NgModule({
-  imports: [DigitalPaymentsModule],
   providers: [
     provideConfig(<I18nConfig>{
+      featureModules: {
+        [CHECKOUT_FEATURE]: {
+          module: () =>
+            import('@spartacus/digital-payments').then(
+              (m) => m.DigitalPaymentsModule
+            ),
+        },
+      },
```

Comment:
I believe there musst be corresponding changes in the installation schematics to support the changes done to this file.